README.md

```diff
@@ -45,11 +45,9 @@ export const wakaq = new WakaQ({
   */
   hardTimeout: Duration.minute(15),
 
-  /* Number of worker processes. Must be an int or str which evaluates to an
-     int. The variable "cores" is replaced with the number of processors on
-     the current machine.
+  /* Number of worker processes. Must be an int.
   */
-  concurrency: 'cores*4',
```

Comment:
Let's leave this, and fix the type of `concurrency` to be `string | number | undefined`

Reply:
Fixed. Ok to merge?